cpe/openscap-cpe-dict.xml

```diff
@@ -137,33 +69,9 @@
             <title xml:lang="en-us">Fedora 33</title>
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.fedora:def:33</check>
       </cpe-item>
-      <cpe-item name="cpe:/o:suse:sle">
-            <title xml:lang="en-us">SUSE Linux Enterprise all versions</title>
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.sle:def:1</check>
-      </cpe-item>
-      <cpe-item name="cpe:/o:suse:sles:10">
-            <title xml:lang="en-us">SUSE Linux Enterprise Server 10</title>
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.sles:def:10</check>
-      </cpe-item>
-      <cpe-item name="cpe:/o:suse:sled:10">
-            <title xml:lang="en-us">SUSE Linux Enterprise Desktop 10</title>
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="openscap-cpe-oval.xml">oval:org.open-scap.cpe.sled:def:10</check>
-      </cpe-item>
-      <cpe-item name="cpe:/o:suse:linux_enterprise_server:11">
```

Comment:
Shouldn't we also change to `cpe:/o:suse:linux_enterprise_server:11:-`?

Reply:
I decided to remove all platforms instead of having spotty support.